src/cryptoadvance/specter/user.py

```diff
@@ -100,6 +100,18 @@ def set_explorer(self, specter, explorer):
         self.config["explorers"][specter.chain] = explorer
         self.save_info(specter)
 
+    def set_proxy_url(self, specter, proxy_url):
```

Comment:
Every user can have his own tor-configuration? Does that really make sense? Isn't that exactly what the admin should be responsible for?